src/style.css

```diff
@@ -0,0 +1,23 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.block {
+  height: 100px;
+  width: 100%;
+  padding: 0;
+  margin: 0;
+}
+
+.block1 {
```

Comment:
don't use numbers to identify the element, use names that describe the content or the behavior of the element